### PR TITLE
ab#1002: bug fix

### PR DESCRIPTION
**Pull request by Anthony-Mckale**
- added "Stopping a docker image" help
- added better name/oerview/recommendation/version for npm audit pypi sca tools
[AB#1002](https://dev.azure.com/riversafe/0b0abc1f-f46a-4d96-a470-e1c546e3cf29/_workitems/edit/1002)

### CODE REVIEW

File: eze/plugins/tools/node_npmaudit.py [hunk 1]
```diff
@@ -130,25 +130,43 @@ def create_recommendation_v7(self, vulnerability: dict):
 
         return recommendation
 
-    def create_path_v7(self, vulnerability: dict):
+    def create_path_v7(self, vulnerability: dict) -> str:
         """convert vulnerability dict into recommendation"""
```
Comment (mgonzales-rs) [inline eze/plugins/tools/node_npmaudit.py:134]: docstring: "extract the path from the vulnerability" ?

File: eze/plugins/tools/node_npmaudit.py [hunk 2]
```diff
@@ -130,25 +130,43 @@ def create_recommendation_v7(self, vulnerability: dict):
...
         for parent_module in vulnerability["effects"]:
             module_path += f"{parent_module}>"
 
+        # pull it all together
+        path = f"{module_path}{module_name}"
+
+        return path
+
+    def create_version_v7(self, vulnerability: dict) -> str:
+        """convert vulnerability dict into recommendation"""
```
Comment (mgonzales-rs) [inline eze/plugins/tools/node_npmaudit.py:149]: docstring: extract the version from the vulnerability dict

File: eze/plugins/tools/node_npmaudit.py [hunk 3]
```diff
@@ -130,25 +130,43 @@ def create_recommendation_v7(self, vulnerability: dict):
...
+        module_version = py_.get(vulnerability, "via[0].range", False) or py_.get(vulnerability, "range")
+
+        return module_version
+
+    def create_description_v7(self, vulnerability: dict):
+        """convert vulnerability dict into recommendation"""
```
Comment (mgonzales-rs) [inline eze/plugins/tools/node_npmaudit.py:155]: docstring - extract the description from the vulnerability dict

**Review by mgonzales-rs — APPROVED** [review]
Running ok: image

File: eze/plugins/tools/node_npmaudit.py (hunk 3 above)
Comment (Anthony-Mckale) [reply eze/plugins/tools/node_npmaudit.py:155]: fixed docstrings , good spot

**Pull request merged**